.github/workflows/main.yml

```diff
@@ -11,10 +11,10 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
-    - name: Set up Python 3.6
+    - name: Set up Python 3.10
       uses: actions/setup-python@v2
       with:
-        python-version: 3.6
+        python-version: 3.10
```

Comment:
BTW I suspect it failed because this is a float and hence 3.1

Reply:
yep yep yep https://github.com/pyvec/pyvo-data/actions/runs/3517306252/jobs/5894923772